src/odemis/driver/andorshrk.py

```diff
@@ -383,12 +386,17 @@ def __init__(self, name: str, role: str,
         if kwargs.get("inverted", None):
             raise ValueError("Axis of spectrograph cannot be inverted")
 
-        if device == "fake":
+        self._uses_shared_dll = (_dll is not None)
```

Comment:
```suggestion
        self._is_dll_shared = (_dll is not None)
```
Upto you which naming is better

Reply:
Sounds like  a better name, indeed. Thanks.